Tailwind app.css drifts when .claude/worktrees/ is present; CI rejects what local accepts

## Symptom

`make tailwind-check` passes locally but fails in CI with:

```
ERROR: internal/web/static/css/app.css is out of date — run `make tailwind` and commit the result.
--- internal/web/static/css/app.css
+++ /tmp/tmp.XXXX
@@ -1,2 +1,2 @@
 /*! tailwindcss v4.3.0 | MIT License | https://tailwindcss.com */
diff: standard output: Resource temporarily unavailable
make: *** [Makefile:223: tailwind-check] Error 1
```

The diff itself overflows the runner's pipe so we can't see what changed. Hit on PR #378 (Bundle A — security) and PR #385 (Bundle H — frontend UX); both required workarounds before they could merge.

## Root cause (current hypothesis)

`make tailwind` on a local clone produced `app.css` ~121 bytes larger than the same command run inside a freshly-cloned copy of the same branch with the same binary (verified via checksum). Diffing the @source directory trees showed identical file content, so Tailwind isn't picking up extra files via the explicit `@source` directives.

Most likely Tailwind v4's auto-content-detection is scanning beyond the explicit `@source` paths and finding files under `.claude/worktrees/` (the per-agent worktree directories the harness leaves behind even after the agents have completed). `.claude/worktrees/` is in `.gitignore`, but Tailwind v4 doesn't always respect `.gitignore` for auto-detection — and even when it does, it'll still walk symlinks the harness creates between the worktree's `_tailwind.css` and the main checkout's CSS layer.

Workaround that unblocked #385: delete `.claude/worktrees/` and re-run `make tailwind`. Output dropped to the clean-clone size and CI accepted it.

## Why it bites

Once the inflated `app.css` is committed (because local `tailwind-check` happily diffs against the same inflated output the worktree-aware binary produces), CI's clean checkout produces the smaller output and refuses to merge. Every PR that touches CSS triggers this; PRs that don't touch CSS hit it whenever the prior committed `app.css` was ever generated with worktrees present.

## Fix options

1. **Add an explicit Tailwind exclude for `.claude/worktrees/`.** Tailwind v4 has `@source not "<glob>"`; add `@source not "../../../../.claude/worktrees/**"` to `_tailwind.css`. Local-friendly: regen still produces stable output regardless of worktree state.
2. **Pre-flight `make tailwind` with a guard** that fails (or scrubs) when `.claude/worktrees/` exists. Less user-friendly — devs running agents would have to clean up before regen.
3. **Switch tailwind-check to use `cmp -s` + a summarised diff** so the "Resource temporarily unavailable" pipe failure stops masking the real content. Doesn't fix the drift, but at least the next time it happens we can see what's drifting.

Option 1 is the durable fix; option 3 is an orthogonal observability improvement worth bundling.

## Out of scope

The bigger CI-vs-local divergence pattern (different runners, different tailwind binaries, different system fonts) is not at play here — binary checksums matched between local and CI.

## Source

Filed after PR #385 (Bundle H) hit the same drift Bundle A's PR #378 was already affected by.

--- a/internal/web/static/css/_tailwind.css
+++ b/internal/web/static/css/_tailwind.css
@@ -26,6 +26,13 @@
 @source "../js";
 @source "../../../client/static";
 
+/* Exclude per-agent worktrees from Tailwind v4's auto-content-detection.
+ * The harness leaves `.claude/worktrees/<agent>/` checkouts behind even
+ * after agents complete, and Tailwind v4 walks them despite .gitignore —
+ * which inflates app.css with utilities only referenced inside those
+ * stale checkouts and fails tailwind-check in CI (#386). */
+@source not "../../../../.claude/worktrees/**";
+
 @theme {
     /* Palette — same tokens as mockups/admin-redesign-tailwind/quiz-list.html. */
     --color-bg: #0a0a0f;

